sound/hda/hdac_stream.c

```diff
@@ -405,11 +406,20 @@ int snd_hdac_stream_setup_periods(struct hdac_stream *azx_dev)
 {
 	struct hdac_bus *bus = azx_dev->bus;
 	struct snd_pcm_substream *substream = azx_dev->substream;
-	struct snd_pcm_runtime *runtime = substream->runtime;
+	struct snd_compr_stream *cstream = azx_dev->cstream;
+	struct snd_pcm_runtime *runtime = NULL;
+	struct snd_dma_buffer *dma_buffer_p = NULL;
```

Comment:
looks like this initialization might be superfluous for dma_buffer_p?

Reply:
@ranj063 That is correct! Thanks for spotting this out.
Ack.